worth2/templates/main/journal_base.html

```diff
@@ -1,6 +1,6 @@
 {% extends 'base.html' %}
 
-{% block title %}Journal for Session {{ session_num }}{% endblock %}
+{% block title %}Journal for Session {{ session_num }} | Participant: {{ participant.study_id }}{% endblock %}
```

Comment:
`study_id` isn't internal, it's the id generated by the study and used by the facilitators.

The purpose for this change is so facilitators know which journal to give to which participant when they print them out.